ui/js/controllers/logviewer.js

```diff
@@ -39,8 +39,17 @@ logViewerApp.controller('LogviewerCtrl', [
                 return;
             }
             $scope.showSuccessful = !$scope.hasFailedSteps();
+            $scope.getSelectedLines();
         });
 
+        $scope.getSelectedLines = function() {
+            var urlHash = $location.hash();
+            var regex = /L(\d+)(-L(\d+))?$/;
+            var match = urlHash.match(regex);
+            $scope.selectedBegin = match[1];
```

Comment:
I'm getting a console error in the browser loading a log with this PR/branch, and I think it claims to be on this line.

```
Error: match is null
Logviewer/$scope.getSelectedLines@http://localhost:8000/js/controllers/logviewer.js:49:13
```

It's generally a good habit to keep the console open during development, sometimes things crop up even though the app may seem to be behaving as you'd expect.